### PR TITLE
minor changes for language.lua

### PR DESCRIPTION
**Pull request by VladWinner**

### CODE REVIEW

File: data/ui_scripts/settings/language.lua [hunk 1]
```diff
@@ -3,19 +3,19 @@ game:addlocalizedstring("MENU_french", "Français")
...
 game:addlocalizedstring("MENU_portuguese", "Português")
 game:addlocalizedstring("MENU_japanese_full", "日本")
 game:addlocalizedstring("MENU_japanese_partial", "日本 (Partial)")
 game:addlocalizedstring("MENU_traditional_chinese", "中国人")
 game:addlocalizedstring("MENU_simplified_chinese", "简体中文")
 game:addlocalizedstring("MENU_arabic", "عربى")
-game:addlocalizedstring("MENU_czech", "Czech") -- ??
-game:addlocalizedstring("MENU_spanishna", "Español (2)") -- ??
+game:addlocalizedstring("MENU_czech", "České") -- ??
+game:addlocalizedstring("MENU_spanishna", "Español (parcial)") -- text only (English dubbing)
```
Comment (fedddddd) [inline data/ui_scripts/settings/language.lua:15]: Uppercase first letter in parenthesis

File: data/ui_scripts/settings/language.lua [hunk 2]
```diff
@@ -3,19 +3,19 @@ game:addlocalizedstring("MENU_french", "Français")
...
 game:addlocalizedstring("MENU_korean", "한국어")
 game:addlocalizedstring("MENU_english_safe", "English (Safe)")
-game:addlocalizedstring("MENU_russian_partial", "Pyccкий (Partial)")
+game:addlocalizedstring("MENU_russian_partial", "Русский (только текст)") -- text only (English dubbing)
```
Comment (fedddddd) [inline data/ui_scripts/settings/language.lua:18]: Same here

**Review by fedddddd — CHANGES_REQUESTED** [review]

File: data/ui_scripts/settings/language.lua (hunk 1 above)
Comment (VladWinner) [reply data/ui_scripts/settings/language.lua]: Fixed.

File: data/ui_scripts/settings/language.lua (hunk 2 above)
Comment (VladWinner) [reply data/ui_scripts/settings/language.lua]: Fixed.

**Pull request merged**